network-api/networkapi/buyersguide/admin.py

```diff
@@ -1,8 +1,9 @@
-from django import forms
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
```

Comment:
Strings from this page, as they are only visible in the CMS, shouldn’t be localized through Pontoon, so you will want to either hardcode them, or add this page to the excluded files from localization https://github.com/mozilla/foundation.mozilla.org/blob/master/tasks.py#L23-L29

Reply:
I'm marking this PR as [demo]: this is not a true PR

Reply:
Sorry for jumping the gun!